include/ipmitool/ipmi_fru.h

```diff
@@ -127,13 +127,13 @@ struct fru_area_product {
 #pragma pack(1)
 #endif
 struct fru_multirec_header {
-#define FRU_RECORD_TYPE_POWER_SUPPLY_INFORMATION 0x00
-#define FRU_RECORD_TYPE_DC_OUTPUT 0x01
-#define FRU_RECORD_TYPE_DC_LOAD 0x02
-#define FRU_RECORD_TYPE_MANAGEMENT_ACCESS 0x03
-#define FRU_RECORD_TYPE_BASE_COMPATIBILITY 0x04
-#define FRU_RECORD_TYPE_EXTENDED_COMPATIBILITY 0x05
-#define FRU_RECORD_TYPE_OEM_EXTENSION	0xc0
+#define FRU_RECORD_TYPE_POWER_SUPPLY_INFORMATION	0x00
+#define FRU_RECORD_TYPE_DC_OUTPUT			0x01
+#define FRU_RECORD_TYPE_DC_LOAD				0x02
+#define FRU_RECORD_TYPE_MANAGEMENT_ACCESS		0x03
+#define FRU_RECORD_TYPE_BASE_COMPATIBILITY		0x04
+#define FRU_RECORD_TYPE_EXTENDED_COMPATIBILITY		0x05
+#define FRU_RECORD_TYPE_OEM_EXTENSION			0xc0
```

Comment:
Although I do like how it looks, I know that this will lead in future to further reformatting of the whole block, which will hinder git-blaming it. Hence, there is a rule to not use pretty formatting:
https://github.com/ipmitool/ipmitool/wiki/Coding-Standards#pretty-formatting